docker-compose.yml

```diff
@@ -1,10 +1,12 @@
 version: '3.7'
+
 services:
   server:
-    container_name: shoe_project_server
     build:
       context: .
       dockerfile: $PWD/docker/Dockerfile.dev
+    container_name: shoe_project_server
+    image: shoe_project_image
```

Comment:
can we make sure that `server` depends on `postgres` as well here? Might as well include in this PR tbh.

Reply:
sure can do